module-2/visualizing-real-world-data/your-code/renewables.ipynb

```diff
@@ -1274,7 +1274,7 @@
     {
```

Comment:
- Try to adjust the legend location on your plots in order to not cover the info in your plot. 
- Would you like to take a look on the error thrown by the following chunk of code?
```Python
# df_violin = dfz[["Windpower", "Solarpower", 'Hydropower', 'Country', 'Year']]
# g = sns.catplot(x="Year", y="Solarpower", hue="Solarpower", data=df_violin, kind="violin")
dfz = df[df['Year']>2010]
g = sns.catplot(x="Year", y="Windpower", hue="Year", data=dfz, logy=True)
```